sdk/appconfiguration/azure-data-appconfiguration/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Release History
+## 1.0.2 (2020-03-09)
+- Updated javadoc to support the changes that App Configuration service no longer support `*a*` and `*a` setting filter.
+  For more information: see [Filtering](https://github.com/Azure/AppConfiguration/blob/master/docs/REST/kv.md#filtering).
```

Comment:
probably want a link to the specific commit/version because /master will change over time.